CITATION.cff

```diff
@@ -0,0 +1,30 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: HERMES Aggregated Interface Script
```

Comment:
I am for having this file, we should lead by example. But, should this not be added in a separate PR? Seems unrelated to this one.

Reply:
Good point. Will move it to another PR :)